server/routes/journeys/screen/confirmation/tests.cy.ts

```diff
@@ -50,10 +50,33 @@ context('test /screen/confirmation', () => {
       },
       csipRecord: {
         status: 'PLAN_PENDING',
+        createdAt: '',
+        createdBy: '',
+        createdByDisplayName: '',
+        prisonNumber: '',
+        recordUuid: '',
+        referral: {
+          contributoryFactors: [],
+          incidentDate: '',
+          incidentLocation: {
+            code: 'A',
+          },
+          incidentType: {
+            code: 'A',
+          },
+          isSaferCustodyTeamInformed: 'DO_NOT_KNOW',
+          refererArea: {
+            code: 'A',
+          },
+          referredBy: '',
+        },
```

Comment:
Can get rid of these changes too for similar reasons above, I've got a patch PR open to fix the inconsistency on this page

Reply:
@JackReeveMoJ this one already had the csipRecord status set in it like so:
```js
csipRecord: {
    status: 'PLAN_PENDING',
},```

 - am I ok to just remove the entire csipRecord, or is the test using that status?

Reply:
you can remove both this and prisoner. they are populated by wiremock stubs.

Reply:
PRs just gone in to correct this, I'd say revert all changes to this file and leave as-is